vendor/VREffect.js

```diff
@@ -232,7 +232,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var cameraR = new THREE.PerspectiveCamera();
 	cameraR.layers.enable( 2 );
 
-	this.render = function ( scene, camera ) {
+	this.render = function ( scene, camera, renderTarget, forceClear ) {

```

Comment:
We should not be changing the code in the vendor folder. If there's something to be changed in VREffect we should submit a PR to three.js https://github.com/mrdoob/three.js/blob/master/examples/js/effects/VREffect.js